modules/utxo-lib/src/bitgo/zcash/ZcashTransaction.ts

```diff
@@ -94,7 +94,7 @@ export class ZcashTransaction extends UtxoTransaction {
     this.consensusBranchId = consensusBranchId ?? getDefaultConsensusBranchIdForVersion(network, this.version);
   }
 
-  static fromBuffer(buffer: Buffer, __noStrict: boolean, network?: ZcashNetwork): ZcashTransaction {
+  static fromBufferZcash(buffer: Buffer, __noStrict: boolean, network?: ZcashNetwork): ZcashTransaction {
```

Comment:
Method removal. Please record as BREAKING CHANGE in commits.

Reply:
refactoring